documentation/misc/Update_Dependencies_Lotus.md

```diff
@@ -0,0 +1,53 @@
+# Updating Lotus Dependencies
+
+This guide will walk through how to update the most common dependencies in Lotus. These are the dependencies this guide currently covers:
+
+- [Filecoin-FFI](#updating-filecoin-ffi)
+- [Ref-FVM](#updating-ref-fvm)
+- [Go-State-Types](#updating-go-state-types)
+- [Builtin-Actors](#updating-builtin-actors)
+
+## Updating Filecoin-FFI
+
+1. In your `/lotus` folder, run `cd extern/filecoin-ffi`.
+
+2. `git pull` to ensure you have the latests changes for *filecoin-ffi*.
+
+3. `git checkout vX.XX.X` to checkout the version you want to update to.
+
+4. Then commit the update to your Lotus branch and open a PR for updating Filecoin-FFI.
+
+👉 Example of a [PR updating Filecoin-FFI](https://github.com/filecoin-project/lotus/pull/11431)
+
+👉 If you need to create a Filecoin-FFI release, you can follow the steps outlined here: [Creating a Filecoin-FFI Release](https://github.com/filecoin-project/filecoin-ffi/pull/461)
+
+## Updating Ref-FVM
```

Comment:
Move this above "Updating Filecoin-FFI" since it potentially happens before that step?

Reply:
Updated in: https://github.com/filecoin-project/lotus/pull/12107/commits/5ea960de0022ca42cb6e213a10ff4ba30c655d13